packages/affiliates/gas-rebate/VoterGasRebate.js

```diff
@@ -634,9 +634,9 @@ async function Main(callback) {
       console.log(`- Using start date: ${moment.unix(startDate).toString()}`);
       console.log(`- Using end date: ${moment.unix(endDate).toString()}`);
 
-      endBlock = (await FindBlockAtTimestamp._findBlockNumberAtTimestamp(web3, Number(endDate), 15, 15, 4, 14))
+      endBlock = (await FindBlockAtTimestamp._findBlockNumberAtTimestamp(web3, Number(endDate), 1, 1, 1, 14))
```

Comment:
Wasn't the default for the higher limit 15 before? Either way, I think this is more precise, but will probably kill our compute time. But this is good for now as it gives us the accuracy back.

Reply:
empirically the compute time is fine setting it as is!